### PR TITLE
LDP-572: Fix tests & scripts & add missing env

### PR DESCRIPTION
**Pull request by petarbasic**

### CODE REVIEW

File: .defaults.env [hunk 1]
```diff
@@ -7,6 +7,7 @@ PERSISTENT_FILES_DIR=files
 URL_SCHEME=https
 APP_SECRET={{ secret }}
 APP_SECRET_LONG={{ secret_long }}
+BEHAT_WARMUP_REQUIRED_CONTENT=window.__NUXT__
```
Comment (fago) [inline .defaults.env:10]: sort of depends on the project, but ok let's assume our decoupled stack here.

File: dotenv/lupus-ci.env [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# @see web/sites/sites.php
+
+# Make use of docker compose environment variables defined in .env.
+APP_MULTISITE_DOMAIN=$COMPOSE_HOSTNAME
+# Set cookie_domain service parameter via services_env_parameter module.
+DRUPAL_SERVICE_session__storage__options___cookie_domain=".ci2.drunomics.com"
+
+# ENV_UNIX_GROUP_WEBSERVER="www-data"
+CHROME_HOST=chrome
+BEHAT_EXTRA_ARGUMENTS="--format=pretty --out=std --format=junit --out=behat_results"
```
Comment (fago) [inline dotenv/lupus-ci.env]: hm, no that file is for internal setup and thus should be added by project-devsetup

File: phpcs.xml.dist [hunk 3]
```diff
@@ -8,19 +8,28 @@
   <file>./scripts</file>
   <file>./web/modules/custom</file>
 
+  <!-- Exclude -->
+  <exclude-pattern>*md</exclude-pattern>
   <exclude-pattern>./scripts/composer/ScriptHandler.php</exclude-pattern>
+  <exclude-pattern>./scripts/ldp-project-kickstart</exclude-pattern>
```
Comment (fago) [inline phpcs.xml.dist]: let's just exclude the whole directory here

**Review by fago — CHANGES_REQUESTED** [review]

File: dotenv/lupus-ci.env (hunk 2 above)
Comment (petarbasic) [reply dotenv/lupus-ci.env]: Good point, already over there.

**Pull request merged**